src/API/Site/Controllers/MerchantCenter/SettingsController.php

```diff
@@ -71,6 +90,9 @@ protected function get_settings_endpoint_edit_callback(): callable {
 			}
 
 			foreach ( $schema as $key => $property ) {
+				if ( ! in_array( 'edit', $property['context'] ?? [], true ) ) {
```

Comment:
I added this to so it doesn't save only view properties; otherwise, it would save them as options, like the `shipping_rates_count` property.